src/equations/compressible_euler_3d.jl

```diff
@@ -1237,6 +1237,97 @@ end
                    u[5])
 end
 
+@inline function (dissipation::DissipationMatrixWintersEtal)(u_ll, u_rr,
+                                                             normal_direction::AbstractVector,
+                                                             equations::CompressibleEulerEquations3D)
+    (; gamma) = equations
+
+    # Step 1:
+    # Rotate solution into the appropriate direction
+
+    norm_ = norm(normal_direction)
+    # Normalize the vector without using `normalize` since we need to multiply by the `norm_` later
+    normal_vector = normal_direction / norm_
+
+    # Some vector that can't be identical to normal_vector (unless normal_vector == 0)
+    tangent1 = SVector(normal_direction[2], normal_direction[3], -normal_direction[1])
+    # Orthogonal projection
+    tangent1 -= dot(normal_vector, tangent1) * normal_vector
+    tangent1 = normalize(tangent1)
+
+    # Third orthogonal vector
+    tangent2 = normalize(cross(normal_direction, tangent1))
+
+    u_ll_rotated = rotate_to_x(u_ll, normal_vector, tangent1, tangent2, equations)
+    u_rr_rotated = rotate_to_x(u_rr, normal_vector, tangent1, tangent2, equations)
+
+    # Step 2:
+    # Compute the averages using the rotated variables
+    rho_ll, v1_ll, v2_ll, v3_ll, p_ll = cons2prim(u_ll_rotated, equations)
+    rho_rr, v1_rr, v2_rr, v3_rr, p_rr = cons2prim(u_rr_rotated, equations)
+
+    b_ll = rho_ll / (2 * p_ll)
+    b_rr = rho_rr / (2 * p_rr)
+
+    rho_log = ln_mean(rho_ll, rho_rr)
+    b_log = ln_mean(b_ll, b_rr)
+    v1_avg = avg(v1_ll, v1_rr)
+    v2_avg = avg(v2_ll, v2_rr)
+    v3_avg = avg(v3_ll, v3_rr)
+    p_avg = avg(rho_ll, rho_rr) / (2 * avg(b_ll, b_rr))
```

Comment:
```suggestion
    v1_avg = 0.5f0 * (v1_ll + v1_rr)
    v2_avg = 0.5f0 * (v2_ll + v2_rr)
    v3_avg = 0.5f0 * (v3_ll + v3_rr)
    p_avg = 0.5f0 * (rho_ll + rho_rr) / (b_ll + b_rr)
```
Make this more explicit and remove the need for the helper function `avg`